renovate.json

```diff
@@ -1,12 +1,18 @@
 {
-  "extends": [
-    "config:base",
-    "helpers:pinGitHubActionDigests"
-  ],
-  "packageRules": [
-    {
-      "packageNames": ["org.mockito:mockito-core"],
-      "schedule": ["on monday"]
-    }
-  ]
+   "extends":[
+      "config:base",
+      "helpers:pinGitHubActionDigests"
+   ],
+   "packageRules":[
+      {
+         "matchUpdateTypes":[
+            "minor",
+            "patch",
+            "pin",
+            "digest"
+         ],
+         "automerge":true,
+         "semanticCommitType":":arrow_up:"
```

Comment:
This needs to be adapted to spoon's commit conventions I guess?

Reply:
https://docs.renovatebot.com/semantic-commits/#semantic-commit-messages Given this doc I should remove this line completely tbh